macro/run_simScript.py

```diff
@@ -201,8 +201,8 @@
  print("cannot have HNL and SUSY or DP at the same time, abort")
  sys.exit(2)
 
-if (simEngine == "Genie" or simEngine == "nuRadiography") and defaultInputFile:
-  inputFile = "/eos/experiment/ship/data/GenieEvents/genie-nu_mu.root"
+# if (simEngine == "Genie" or simEngine == "nuRadiography") and defaultInputFile:
+#   inputFile = "/eos/experiment/ship/data/GenieEvents/genie-nu_mu.root"
```

Comment:
This seems to be unrelated to the muon shield.
If you want to propose changes to the neutrino simulation, please make a separate pull request for that.